data_tfms.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">trial = pd.read_json("data/raw/trial.json")
train = pd.read_json("data/raw/train.json")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>PR it when you are updating paths? PLIJ PLIJ PLIZ</p>

Reply:
Resolved by 6885478